keystorage/aws/src/main/java/tech/pegasys/signers/aws/AwsSecretsManager.java

```diff
@@ -68,6 +76,39 @@ static AwsSecretsManager createAwsSecretsManager() {
     }
   }
 
+  public <R> Collection<R> mapSecretsList(
```

Comment:
Just mapSecrets then it would be consistent with what we are using in AzureKeyVault